pkgs/applications/version-management/sr-ht/core.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, fetchgit, fetchNodeModules, buildPythonPackage
+, flask, flask_login, humanize, sqlalchemy, sqlalchemy-utils, psycopg2, markdown
+, bleach, requests, beautifulsoup4, pgpy, misaka, pygments
+, sassc, nodejs-11_x
+, pytest, python, alembic, writeText }:
+
+buildPythonPackage rec {
+  pname = "core-sr-ht";
+  version = "0.30.5";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~sircmpwn/core.sr.ht";
+    rev = version;
+    sha256 = "0ccbymmixldf0mb8dmcij2j7v0cxp1vhrfbl6az1hv5y1rxijqb6";
+  };
+
+  node_modules = fetchNodeModules {
+    src = "${src}/srht";
+    nodejs = nodejs-11_x;
+    sha256 = "0axl50swhcw8llq8z2icwr4nkr5qsw2riih0a040f9wx4xiw4p6p";
+  };
+
+  patches = [
+    ./disable-npm-install.patch
+  ];
+
+  buildInputs = [
+    sassc
+    nodejs-11_x
+  ];
+
+  propagatedBuildInputs = [
+    flask
+    flask_login
+    humanize
+    sqlalchemy
+    sqlalchemy-utils
+    psycopg2
+    markdown
+    bleach
+    requests
+    beautifulsoup4
+    pgpy
+    misaka
+    pygments
+
+    # Unofficial runtime dependency?
+    alembic
+  ];
+
+  PKGVER = version;
+
+  preBuild = ''
+    cp -r ${node_modules} srht/node_modules
+  '';
+
+  # No actual? tests but seems like it needs this anyway
+  preCheck = let
```

Comment:
I have no clue what's going on here but the `checkPhase` succeeds now so that's good?